src/components/Sidebar/index.js

```diff
@@ -132,6 +132,7 @@ Sidebar.propTypes = {
   children: PropTypes.arrayOf(function(propValue, key) {
     for (let i = 0; i < propValue.length; i++) {
       if (
+        typeof propValue[i] !== 'boolean' &&
```

Comment:
can you explain this?

Reply:
basically whenever we do `(something) &&  (component)` in the jsx it will pass `false` as a child to the sidebar if something is `false`.

Reply:
makes sense - and weird. :D can you add a comment plz? @c-o-l-o-r